src/components/scenes/FioAddressRegisterScene.js

```diff
@@ -204,9 +204,6 @@ export class FioAddressRegisterScene extends React.Component<Props, State> {
         } catch (e) {
           if (!e.labelCode || e.name !== 'FioError') {
             showError(s.strings.fio_connect_wallets_err)
-            return this.setState({
```

Comment:
I think you needed to add:
'isAvailable: false'
to setState instead of removing it no?

Reply:
no, there is setState below
```
if (!e.labelCode || e.name !== 'FioError') {
            showError(s.strings.fio_connect_wallets_err)
          }
          return this.setState({
            isAvailable: false,
            loading: false
          })
```